lib/issuer.js

```diff
@@ -188,33 +230,206 @@ api.claim = function(actor, id, callback) {
   });
 };
 
-bedrock.events.on('bedrock-mongodb.ready', function(callback) {
-  async.auto({
-    createIndexes: function(callback) {
-      // create indexes for claimed status
-      database.createIndexes([{
-        collection: store.provider.name,
-        fields: {recipient: 1, 'credential.sysState': 1, id: 1},
-        options: {unique: true, background: false}
-      }, {
-        collection: store.provider.name,
-        fields: {issuer: 1, 'credential.sysState': 1, id: 1},
-        options: {unique: true, background: false}
-      }], callback);
+// TODO: more acceptable formats?
+function checkHeaders(req, res, next) {
+  var acceptableContentTypes = [
+    'application/json',
+    'application/ld+json',
+    'text/html'
+  ];
+  if(!req.accepts(acceptableContentTypes)) {
+    res.status(406);
+    res.write(JSON.stringify({
+      message: 'Requested content types not acceptable.',
+      details: {
+        accepted: req.get('accept'),
+        acceptable: acceptableContentTypes
+      }
+    }));
+    res.end();
+    return next();
+  }
+  if(acceptableContentTypes.indexOf(req.get('Content-Type')) == -1) {
```

Comment:
I'll have to revisit this again, but I believe it's checking the content type that has been provided as well as what is expected in return.